Sources/Clappr/Classes/Extension/Dictionary+Ext.swift

```diff
@@ -14,3 +14,12 @@ extension Dictionary where Key == String, Value == Any {
         }
     }
 }
+
+public extension Dictionary where Key == String, Value == Any {
+    func bool(_ option: String) -> Bool {
```

Comment:
```suggestion
    func bool(_ option: String, or alternative: Bool = false)
```
so that when using, we can make it explicit what happens in case the option is not present. For example, inside `AVFoundationPlayback.shouldLoop` you could write `options.bool(kLoop, or:false)`

Reply:
I like that 👏